web/Viewer/SyncronusScrolling/SyncronusScrolling-PSPDFKit-main/Readme.md

```diff
@@ -0,0 +1,13 @@
+# About project
+
+Load two instances/viewers and synchronous scrolling/zooming of the right-side viewer based on the left-side viewer
+
+    clone the repository
+    cd to the repository
+    npm install
+    cp -R ./node_modules/pspdfkit/dist/pspdfkit-lib/ assets/pspdfkit-lib
+    cp -R ./node_modules/pspdfkit/dist/pspdfkit.js assets/
+
+    Add the PSPDFKIt license key in index.html
+    either using the live server or web server run the index.html
+    

```

Comment:
The project itself looks good, but I'd rework the readme on how to run the project a bit, similary to how it looks here: https://github.com/PSPDFKit/awesome-pspdfkit/pull/29/files#diff-16cc5e6c0873276d5aacef903497e8d244715e5f4d02aa07d3f9ab1b158ecd02

Reply:
@ChrisixFlash : are you updating the readme and correcting the spelling ?